serverless_k8s/README.md

```diff
@@ -0,0 +1,73 @@
+# Serverless Kubernetes Track
+Serverless Kubernetes is a track intended for benchmarking serverless Elasticsearch using a Kubernetes pod metrics dataset. A small corpus of Kibana object definitions is included for indexing with fast refresh enabled.
+
```

Comment:
I'm not sure how I feel about the name, will this ever be ran against stateful ES? Is it enough to just name it something related to Search After Write, or similar?